subcommand/server-acl-init/rules.go

```diff
@@ -240,26 +243,37 @@ namespace_prefix "" {
 	return c.renderRules(aclReplicationRulesTpl)
 }
 
-// todo: implement namespaces support.
 func (c *Command) controllerRules() (string, error) {
 	controllerRules := `
 operator = "write"
-node_prefix "" {
-  policy = "write"
+{{- if .EnableNamespaces }}
+{{- if .InjectEnableNSMirroring }}
+namespace_prefix "{{ .InjectNSMirroringPrefix }}" {
+{{- else }}
+namespace "{{ .InjectConsulDestNS }}" {
+{{- end }}
+  service_prefix "" {
+    policy = "write"
+  }
 }
+{{- else }}
 service_prefix "" {
   policy = "write"
 }
+{{- end }}
```

Comment:
Cant we just call the `{{- end }}` block before we write the `service_prefix` and we wouldnt have to duplicate that snippet of the config?

Reply:
Yeah so the other ones do that but it results in the formatting having an extra two spaces in OSS:
```
operator = "write"
  service_prefix "" {
    policy = "write"
  }
```

vs

```
operator = "write"
service_prefix "" {
  policy = "write"
}
```

This shows up in the Consul UI and always kinda bothered me. What do you think though?

Reply:
👀 The formatting on the first one hurts my soul. I'll resolve this because that formatting bothers me as well.

Reply:
It is inconsistent to only do it here though... I'll change to be like the rest and add a note on our backlog.